client/asset/eth/eth_test.go

```diff
@@ -1085,58 +941,9 @@ func TestSwap(t *testing.T) {
 					testName, receipt.Contract(), contract.SecretHash)
 			}
 
-			if !bytes.Equal(node.lastInitiation.hash.Bytes(), receipt.Coin().ID()) {
```

Comment:
Why did you remove these? Shouldn't we be checking that `initiate` is called with correct arguments?

Reply:
Good call. You probably see the PR, but I'll link for all: https://github.com/decred/dcrdex/pull/1318